libcloud/common/digitalocean.py

```diff
@@ -109,6 +109,16 @@ def add_default_headers(self, headers):
         headers['Content-Type'] = 'application/json'
         return headers
 
+    def add_default_params(self, params):
+        """
+        Add parameters that are necessary for every request
+
+        This method adds ``per_page`` to the request to reduce the total
+        number of paginated requests to the API.
+        """
+        params['per_page'] = '200'
```

Comment:
It would be nice if user could also specify / override this value - maybe by passing an argument to the driver constructor?